policyengine_us/parameters/gov/states/wi/tax/income/credits/childcare_expense/fraction.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
       href: https://www.revenue.wi.gov/TaxForms2022/2022-Form1-Inst.pdf#page=17
     - title: 2023 Form 1 instructions
       href: https://www.revenue.wi.gov/TaxForms2023/2023-Form1-Inst.pdf#page=18
+    - title: 2024 2023 WISCONSIN ACT SECTION 2. 71.07 (9g) (b) 2.
+      href: https://content.govdelivery.com/attachments/WIGOV/2024/03/04/file_attachments/2802082/ab1023.pdf
```

Comment:
Replace with a legal code source.

If the legal code does not yet incorporate this increase, additionally add a source indicating this bill was signed